libpng/PSPBUILD

```diff
@@ -9,37 +9,23 @@ license=('ZLIB')
 depends=('zlib')
 makedepends=()
 optdepends=()
-source=(
-    "http://sourceforge.net/projects/libpng/files/libpng15/older-releases/${pkgver}/${pkgname}-${pkgver}.tar.gz"
-    "${pkgname}-${pkgver}-PSP.patch"
-)
-sha256sums=(
-    '7d6431388f67b48f20242d2623e1aaa99e1c7cab5d2d699742d0e4ac987e8241'
-    'SKIP'
-)
-
-prepare() {
-    cd "$pkgname-$pkgver"
-    patch -p1 < ../${pkgname}-${pkgver}-PSP.patch
-}
+source=("git+https://github.com/glennrp/libpng.git#commit=a40189cf881e9f0db80511c382292a5604c3c3d1")
```

Comment:
Please use the official versioned tar.gz or add the commit to the version number